tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Validators/OrganisationValidatorTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NHSD.BuyingCatalogue.Organisations.Api.Models;
+using NHSD.BuyingCatalogue.Organisations.Api.Repositories;
+using NHSD.BuyingCatalogue.Organisations.Api.UnitTests.Builders;
+using NHSD.BuyingCatalogue.Organisations.Api.Validators;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.UnitTests.Validators
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    internal sealed class OrganisationValidatorTests
+    {
+        [Test]
+        public void Constructor_NullRepository_Throws()
+        {
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                var _ = new OrganisationValidator(null);
+            });
+        }
+
+        [Test]
+        public async Task Validate_Organisation_Does_Not_Exist_Returns_Success()
+        {
+            var newOrganisation = OrganisationBuilder.Create(1).Build();
+
+            var mockOrganisationRepository = SetUpGetByOdsCodeAsync(null);
+
+            var validator = new OrganisationValidator(mockOrganisationRepository);
+
+            var result = await validator.ValidateAsync(newOrganisation);
+
+            result.IsSuccess.Should().BeTrue();
```

Comment:
It makes the code easier to read and understand when there is just one assert function per test. This suggestion could be applied here e.g. actual.Should().Be(Result.Success());

This also highlights the use of the 'actual' variable, that I believe we said in the coding standards meeting we would introduce (along with 'expected' but probably not worth it here).

Reply:
👍 Agreed & did something similar with the failure test.